setup.py

```diff
@@ -61,7 +61,7 @@
 SKIP_CUDA_BUILD = os.getenv("FLASH_ATTENTION_SKIP_CUDA_BUILD", "FALSE") == "TRUE"
 # For CI, we want the option to build with C++11 ABI since the nvcr images use C++11 ABI
 FORCE_CXX11_ABI = os.getenv("FLASH_ATTENTION_FORCE_CXX11_ABI", "FALSE") == "TRUE"
-
+USE_TRITON_ROCM = os.getenv("FLASH_ATTENTION_USE_TRITON_ROCM", "FALSE") == "TRUE"
```

Comment:
If `USE_TRITON_ROCM` is true, we can also skip the `git submodule update` command  at line 137, as well as as the `os.system` calls  to generate code from the composable kernel repo, line 309.

Reply:
done